src/test/java/microsoft/sql/DateTimeOffsetTest.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Microsoft JDBC Driver for SQL Server Copyright(c) Microsoft Corporation All rights reserved. This program is made
+ * available under the terms of the MIT License. See the LICENSE file in the project root for more information.
+ */
+
+package microsoft.sql;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;

```

Comment:
nice to have, but we don't do this in the other tests so to keep things consistent, pls remove